### PR TITLE
luci-app-upnp: cleanup defaults formatting

### PR DESCRIPTION
**Pull request by atiensivu**
Reformat s.default to coding guidelines.

### CODE REVIEW

**Comment by feckert** [comment]
@atiensivu is this a duplicate https://github.com/openwrt/luci/pull/4277 ?

File: applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js [hunk 1]
```diff
@@ -121,14 +121,19 @@ return view.extend({
 		o = s.taboption('general', form.Flag, 'enabled', _('Start UPnP and NAT-PMP service'));
 		o.rmempty  = false;
 
-		s.taboption('general', form.Flag, 'enable_upnp', _('Enable UPnP functionality')).default = '1'
-		s.taboption('general', form.Flag, 'enable_natpmp', _('Enable NAT-PMP functionality')).default = '1'
+		s.taboption('general', form.Flag, 'enable_upnp', _('Enable UPnP functionality'))
+		s.default = '1'
```
Comment (jow-) [inline applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js:125]: This needs to be
```
o = s.taboption(...);
o.default = 1;
```

File: applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js [hunk 2]
```diff
@@ -121,14 +121,19 @@ return view.extend({
...
+
+		s.taboption('general', form.Flag, 'enable_natpmp', _('Enable NAT-PMP functionality'))
+		s.default = '1'
```
Comment (jow-) [inline applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js:128]: dito

File: applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js [hunk 3]
```diff
@@ -121,14 +121,19 @@ return view.extend({
...
 
 		s.taboption('general', form.Flag, 'secure_mode', _('Enable secure mode'),
-			_('Allow adding forwards only to requesting ip addresses')).default = '1'
+			_('Allow adding forwards only to requesting ip addresses'))
+		s.default = '1'
```
Comment (jow-) [inline applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js:132]: dito

File: applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js [hunk 4]
```diff
@@ -121,14 +121,19 @@ return view.extend({
...
 
 		s.taboption('general', form.Flag, 'igdv1', _('Enable IGDv1 mode'),
-			_('Advertise as IGDv1 device instead of IGDv2')).default = '0'
+			_('Advertise as IGDv1 device instead of IGDv2'))
+		s.default = '0'
```
Comment (jow-) [inline applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js:136]: dito

File: applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js [hunk 5]
```diff
@@ -143,7 +148,8 @@ return view.extend({
 		o.datatype = 'port'
 		o.default  = 5000
 
-		s.taboption('advanced', form.Flag, 'system_uptime', _('Report system instead of daemon uptime')).default = '1'
+		s.taboption('advanced', form.Flag, 'system_uptime', _('Report system instead of daemon uptime'))
+		s.default = '1'
```
Comment (jow-) [inline applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js:152]: dito

File: applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js [hunk 6]
```diff
@@ -167,6 +173,18 @@ return view.extend({
 		o = s.taboption('advanced', form.Value, 'upnp_lease_file', _('UPnP lease file'))
 		o.placeholder = '/var/run/miniupnpd.leases'
 
+		s.taboption('advanced', form.Flag, 'use_stun', _('Use STUN'))
+		o.default = '0'
```
Comment (jow-) [inline applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js:177]: dito

**Comment by feckert** [comment]
Please rebase

**Pull request closed without merging**